src/_guides/language/language-tour.md

```diff
@@ -3914,6 +3914,16 @@ main() {
 }
 {% endprettify %}
 
+<aside class="alert alert-info" markdown="1">
+**Version note:**
+A new form of function types was introduced in Dart 1.24. Unless you define
```

Comment:
The second sentence was hard for me to parse. Also, "Version note" didn't necessarily seem like the best title, unless you start the sentence with "Dart 1.24"...but even then, this is more a note about new syntax than about versions. An example would be nice, too. Maybe something like this (I've probably used incorrect wording, but you can fix that!):

---------
**New function type syntax:** Dart 1.24 introduced a new form of function types, the [generic function type alias.](https://github.com/dart-lang/sdk/blob/master/docs/language/informal/generic-function-type-alias.md) You might use this new feature if you pass around generic methods, define fields that are function types, or define arguments with generic function types. Here's an example of using the new syntax:

```
typedef F = List<T> Function<T>(T);
```
---------